src/javascript/EditPanel/EditPanelContent/FormBuilder/FieldSet/FieldSet.jsx

```diff
@@ -18,8 +18,15 @@ let styles = theme => ({
     fieldsetTitleContainer: {
```

Comment:
(Not mandatory) You can use a scss file to allow to remove the HOC "withStyles" to set the style.
We did it for some others components.